Add installation instruction (#526)
tinkoff-ai/etna: README.md

## Installation 

ETNA is available on [PyPI](https://pypi.org/project/etna), so you can use `pip` to install it.

Install default version:
```bash
pip install --upgrade pip
pip install etna
```

Default version hasn't all the dependencies, because some of them are needed only for specific models, e.g. Prophet, PyTorch. 

> Available extensions are listed at [`pyproject.toml`](https://github.com/tinkoff-ai/etna/blob/master/pyproject.toml#L93).

Author: I think it can be difficult to support it after adding new extensions or changing current.

Without the appropriate extension you will get an `ImportError` trying to import the model that needs it.
For example, `etna.models.ProphetModel` needs `prophet` extension and can't be used without it.

Install extension:
```bash
pip install etna[extension-name]

> ```

Author: Even if extensions are listed it can be not obvious how to install them. This instruction makes it clear.

> ```

Reviewer: Maybe list extensions as installation instructions:

```bash
pip install etna[prophet]
pip install etna[torch]
pip install etna[wandb]
```

> ```

Reviewer: If you are against it, let's keep it your way.

Install all extensions:
```bash
pip install etna[all]
```

### Configuration

It can be useful to configure the library to check installed packages during init.

> For example, if you know that you need `Prophet` in you project and don't want to understand that it isn't installed only during the import of `etna.models.ProphetModel`.

Reviewer: Maybe this way it will clearer:

etna supports configuration files. It means that etna will check that all the specified packages are installed prior to script start and NOT during runtime.

> You can create a config file `.etna` at the root of your project, where this checking can be set up. To see the available options look at [`Settings`](https://github.com/tinkoff-ai/etna/blob/master/etna/settings.py#L68). There is an [example](https://github.com/tinkoff-ai/etna/tree/master/examples/configs/.etna) of configuration file. 

Reviewer: ```suggestion
To create a config file you should create `.etna` file at the root of your project. To see the available options look at [`Settings`](https://github.com/tinkoff-ai/etna/blob/master/etna/settings.py#L68). There is an [example](https://github.com/tinkoff-ai/etna/tree/master/examples/configs/.etna) of configuration file. 
```

## Installation 

ETNA is available on [PyPI](https://pypi.org/project/etna), so you can use `pip` to install it.

Install default version:
```bash
pip install --upgrade pip
pip install etna
```

The default version doesn't contain all the dependencies, because some of them are needed only for specific models, e.g. Prophet, PyTorch.
Available user extensions are the following:
* `prophet`
* `torch`
* `wandb`

> There are also developer extensions. All the extensions are listed at [`pyproject.toml`](https://github.com/tinkoff-ai/etna/blob/master/pyproject.toml#L93).

Reviewer: ```suggestion
There are also developer extensions. All the extensions are listed in [`pyproject.toml`](https://github.com/tinkoff-ai/etna/blob/master/pyproject.toml#L93).
```
